js/components/popover.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+<div class="popover" :class="classes" v-show="show" :style="style" :transition="effect">
+    <div class="arrow"></div>
+    <h3 class="popover-title" v-show="title">{{title}}</h3>
+    <div class="popover-content" v-el:content>{{content}}</div>
+</div>
+</template>
+<script>
+import log from 'logger';
+import TooltipMixin from 'mixins/tooltip';
+
+import { popover } from 'vue-strap'; // Needed for style
+
+export default {
+    mixins: [TooltipMixin],
+    props: {
+        // An optionnal title
+        title: [String, Element, HTMLElement],
+        // The popover content
+        content: [String, Element, HTMLElement],
+        // Wider popover
+        large: {type: Boolean, default: false},
+    },
+    computed: {
+        classes() {
+            const classes = {large: this.large};
```

Comment:
Already tried ! Sadly, it's not working :( Each time I want to write `{this.attribute}` instead of `{attribute: this.attribute}` (I'm a lazy developer :)) but it's not working.